marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/CallBatcherImpl.java

```diff
@@ -414,4 +418,50 @@ public Boolean call() {
             }
         }
     }
+
+	@Override
+	public CallBatcherImpl<W, E> withdefaultArgs(CallArgs args) {
+		if(args == null)
+			throw new IllegalArgumentException("CallArgs cannot be null.");
+		CallArgsImpl callArgsImpl = (CallArgsImpl) args;
+		if(callArgsImpl == null || ! (callArgsImpl instanceof CallArgsImpl))
+			throw new IllegalArgumentException("Illegal Argument.");
+		if(callArgsImpl.getEndpoint().getEndpointPath()!= caller.getEndpointPath())
+			throw new IllegalArgumentException("Endpoints are different.");
+		
+		List<CallField> callFieldList = callArgsImpl.getCallFields();
+		callFieldList.forEach(p->p.toBuffered());
```

Comment:
Does this line store the buffered fields anywhere?

Would it be best to create a new list so the existing list owned by the application CallArgs object is not modified?

If the args have a null or empty field, should the withDefaultArgs() method return this without doing anything?